sql/moz-fx-data-shared-prod/telemetry_derived/firefox_desktop_usage_v1/metadata.yaml

```diff
@@ -25,3 +25,12 @@ scheduling:
   # previous day and adding dau from the new day, but this would make each
   # query dependent on history. Recreating the table is simpler and more robust.
   date_partition_parameter: null
+bigquery:
+  time_partitioning:
+    field: submission_date
+    type: day
+    require_partition_filter: false
+  clustering:
+    fields:
+    - attributed
+    - country
```

Comment:
We should confirm that BQ is able to pushdown this predicate through the join, since we're actually filtering on `country_name`

Reply:
If not, we can materialize `country_name` as well.

Reply:
Good callout, and I highly doubt it will be able to take advantage of this. Are we able to push the responsibility of mapping between ISO codes and country names to the Looker layer? It's not clear to me whether the following is applicable to our case:

https://docs.looker.com/reference/field-params/map_layer_name

Reply:
I don't think so - `map_layer_name` is what Looker uses to translate the string to a specific location on a map. I'll check in with Looker support to make sure there's nothing else we can use.

Reply:
Looker folks confirmed there's no hidden functionality for this. We could put in a huge CASE statement, but I'd much rather materialize this in the data than in the query itself :)

Reply:
Will materialize.

Reply:
I've added country_name below. We are clustering on both country and country_name, but that should be essentially free since they're a 1:1 mapping.